style.css

```diff
@@ -3,115 +3,126 @@ body {
   margin: 0;
 }
 
-/* Tag selector for all links and h1 elements {
-  font-family: 'Futura webfont', sans-serif;
-}*/
+/* Tag selector for all links and h1 elements*/
+a, h1 {
+  font-family  : 'Futura webfont', sans-serif;
+}
 
-/* Tag selector for all h1 elements {
-  font-size: 3, 75rem;
-  margin: 0;
-  padding-top: 16px;
-}*/
+/* Tag selector for all h1 elements */
+h1 {
+  font-size    : 3, 75rem;
+  margin       : 0;
+  padding-top  : 16px;
+}
+
+/* Tag selector for all h2 elements and p  */
+h2, p {
+  font-family  : 'Mulish', sans-serif;
+}
+
+/* Tag selector for all p elements */
+p {
+  line-height  : 25px;
+}
 
-/* Tag selector for all h2 elements and p {
-  font-family: 'Mulish', sans-serif;
-} */
 
-/* Tag selector for all p elements {
-  line-height: 25px;
+/* Tag selector for all h2 elements */
+ h2 {
+  font-size    : 1, 65rem;
+  line-height  : 29px;
+  font-weight  : 700;
 }
-*/
 
-/* Tag selector for all h2 elements {
-  font-size: 1, 65rem;
-  line-height: 29px;
-  font-weight: 700;
+
+/* Class selector for a class that should have the BG image */
```

Comment:
/* Class selector for a class that should have the BG image */
`.hero-content {
  background-image: url(https://uploads-ssl.webflow.com/60780bff57ddc42a6adc1d7e/60dad44bbcd8cb866c91a130_Teaching%20Learning%201.jpg);
  background-position: 50% 50%;
  background-size: cover;
  min-height: 500px;
}`